packages/cli/src/__tests__/ceramic-daemon.test.ts

```diff
@@ -384,11 +391,78 @@ describe('Ceramic interop: core <> http-client', () => {
       fetchJson(`http://localhost:${daemon.port}/api/v0/streams/${doc.id}/content`, {
         timeout: 1000,
       })
-    ).rejects.toThrow(`Http request timed out after 1000 ms`)
```

Comment:
I'm frustrated by this loss of context.  Looking at the node docs, it seems like `AbortController.abort` should be able to take an option string arg `reason` which could be used to provide a context string that the AbortSignal will include when it's aborted, and which can be used to give more meaningful error messages when an operation is cancelled.  Sounds great!  But in my testing I'm not actually able to get it to work - I keep getting an error `Expected 0 arguments, but got 1`.

Reply:
Update: I played around with different versions node using nvm.  Node 16.14.0 does indeed have the `reason` arg, but Node 16.13.1 doesn't seem to have it.  So I guess this is a very recent addition. 

But even though I can select node 16.14.0 with NVM and see it working by running `node` manually, when I try to use it in Ceramic I still get a compilation error `Expected 0 arguments, but got 1`.  Any ideas why that would be @ukstv?  Does that mean my Ceramic build is still using Node 16.13.1 for some reason, even though `node --version` is showing 16.14.0?

Reply:
@stbrody 1. `Expected 0 arguments` - if it is in TS, then Node.js types do not include reason in the declarations.
2. I tend to believe, it does not matter if we pass a reason there to the AbortController. [node-fetch](https://github.com/node-fetch/node-fetch/blob/main/src/index.js#L70) does not respect `reason` anyway.

Reply:
We could do something like https://github.com/node-fetch/node-fetch/issues/1462#issuecomment-1059995501 though @stephhuynh18

Reply:
I used to do something like https://github.com/node-fetch/node-fetch/issues/1462#issuecomment-1059995501 but with this change we won't be able to tell if the request aborted because of a timeout or because the user called abort themselves.

Reply:
> I used to do something like [node-fetch/node-fetch#1462 (comment)](https://github.com/node-fetch/node-fetch/issues/1462#issuecomment-1059995501) but with this change we won't be able to tell if the request aborted because of a timeout or because the user called abort themselves.

Not good.

Current code in the PR is indeed the most appropriate.

Reply:
looks like there's a bunch of open issues on various repos to get this more fully supported.  Agreed it's not worth doing anything more on this now, but we should revisit in the future once there's better support for abort reasons throughout the node ecosystem